src/wallet3/default_daemon_comms.cpp

```diff
@@ -239,8 +256,16 @@ namespace wallet
       throw;
     }
 
-    // TODO: proper callbacks
-    conn = omq->connect_remote(remote, [](auto){}, [](auto,auto){});
+    oxen::log::info(logcat, "Daemon Comms trying to connect to OMQ");
```

Comment:
probably better to say "trying to connect to remote oxend" or something like that.